### PR TITLE
change page title to reflect current tab

### PR DESCRIPTION
**Pull request by micahgodbolt**
fixes https://github.com/microsoft/FluidFramework/issues/3813

page title now changes to reflect the tab that the user is on

### CODE REVIEW

File: .storybook/sourcecode-addon/storybookPanel.jsx [hunk 1]
```diff
@@ -35,6 +35,13 @@ const SourceCodePanel = props => {
 
   const handleToggleCompiled = () => setShowCompiled(!showCompiled);
 
+// Update page title on load or tab change
+  useEffect(() => {
+    const title = api.getCurrentStoryData()?.kind + ' ' + api.getUrlState()?.viewMode
```
Comment (tylerbutler) [inline .storybook/sourcecode-addon/storybookPanel.jsx:40]:
```suggestion
    const title = `${api.getCurrentStoryData()?.kind} ${api.getUrlState()?.viewMode}`;
```
nit: I find this more readable - but it's your call.

Comment (tylerbutler) [inline .storybook/sourcecode-addon/storybookPanel.jsx:40]:
```suggestion
    const title = api.getCurrentStoryData()?.kind + ' ' + api.getUrlState()?.viewMode;
```
A simpler fix for the missing semicolon in the case you don't like the other suggestion.

**Review by tylerbutler — APPROVED** [review]

**Comment by micahgodbolt** [comment]
@SamBroner i don't have admin privs to push this through. can you review or promote me?

**Review by SamBroner — APPROVED** [review]
Great

**Comment by SamBroner** [comment]
Seems like I need to fix our build pipeline, not sure why it isn't running.

**Comment by tylerbutler** [comment]
@micahgodbolt Can you address the merge conflict? I can override and merge after that.

**Pull request merged**